src/minimega/web.go

```diff
@@ -201,16 +207,48 @@ func webScreenshot(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+// Redirect / to /vms
 func webIndex(w http.ResponseWriter, r *http.Request) {
 	if r.URL.Path != "/" {
 		http.NotFound(w, r)
-	} else {
-		http.ServeFile(w, r, filepath.Join(web.Root, "index.html"))
+		return
 	}
+
+	http.Redirect(w, r, "/vms", 302)
 }
 
-func webTileVNC(w http.ResponseWriter, r *http.Request) {
-	http.ServeFile(w, r, filepath.Join(web.Root, "tilevnc.html"))
+// Templated HTML responses
+func webTemplated(w http.ResponseWriter, r *http.Request) {
+	lp := filepath.Join(web.Root, "templates", "_layout.tmpl")
+	fp := filepath.Join(web.Root, "templates", r.URL.Path+".tmpl")
+
+	info, err := os.Stat(fp)
+	if err != nil {
+		if os.IsNotExist(err) {
```

Comment:
You can get rid of `if err != nil`: `os.IsNotExist(err)` returns false if `err` is nil. I would probably return 404 if `err != nil`.